worlds/animal_well/client.py

```diff
@@ -170,6 +215,60 @@ def on_package(self, cmd: str, args: dict):
         if cmd == "Connected":
             self.slot_data = args.get("slot_data", {})
 
+        try:
+            def parseIds(data):
+                for item in data:
+                    if 'type' in item and 'text' in item:
+                        if item['type'] == 'player_id':
+                            item['text'] = self.player_names[int(item['text'])]
+                        elif item['type'] == 'item_id':
+                            item['text'] = item_id_to_name[int(item['text'])]
+                        elif item['type'] == 'location_id':
+                            item['text'] = location_id_to_name[int(item['text'])]
+
+                return data
+
+            if cmd == "PrintJSON":
+                msgType = args.get("type")
+                if msgType == "Chat":
+                    display_to_client(self, "".join(o['text'] for o in parseIds(args.get("data"))))
+                elif msgType == "Hint":
+                    text = "".join(o['text'] for o in parseIds(args.get("data")))
+                    display_to_client(self, text)
+                elif msgType == "Join":
+                    display_to_client(self, args.get('data')[0]['text'])
+                elif msgType == "CommandResult":
+                    display_to_client(self, args.get('data')[0]['text'])
+                elif msgType == "Tutorial":
+                    pass
+                elif msgType == "ItemSend":
+                    data = parseIds(args.get('data'))
+                    text = "".join(o['text'] for o in data).replace(')', '').replace('(', 'at ')
+                    display_to_client(self, text)
+                elif msgType == "Countdown":
+                    # {'cmd': 'PrintJSON', 'data': [{'text': '[Server]: GO'}], 'type': 'Countdown', 'countdown': 0}
+                    text = "".join(o['text'] for o in args.get('data'))
+                    display_to_client(self, text)
```

Comment:
We should totally make some dumb little effect happen when the countdown says "GO". I'm not sure what our options are, and this is totally unimportant for this, but it would be cute

Reply:
I could probably find something fun (and stupid (and possibly terrifying)) to implement here.

Reply:
If we can figure out a sprite that would be a good candidate for it, I've tracked down the way that the game triggers showing a symbol above the player (like when you pick up a firecracker), and I could use that to display something like maybe that dice symbol that is used in the flute room to teach you the number->direction connection. Another option could be drawing 3 kangaroos, then 2 kangaroos, then 1 kangaroo, then GO.